experimental/Schemes/WeilDivisor.jl

```diff
@@ -33,6 +33,7 @@ stored as a formal linear combination over some ring ``R`` of
       coefficients::IdDict{<:IdealSheaf, CoefficientRingElemType};
       check::Bool=true
     ) where {CoefficientRingType, CoefficientRingElemType}
+    @check
     if check
```

Comment:
I agree that the standalone check is rather awkward, especially in conjunction with another `if check` - block. I was not aware of the possibility to do a `do` block. It looks way more reasonable.

Reply:
Correction: a `do` block won't work, what I should have suggested is a `begin`-block. I've already communicate this to @HechtiDerLachs via Slack, but I wanted to clarify it here, too, in case I am confusing any other readers. Sorry for that :-(